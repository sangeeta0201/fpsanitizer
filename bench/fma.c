#include <stdio.h>
#include <math.h>

int main() {
  volatile double x1, y1, z1;
  double x, y, z, w;
  x1 = 1 + 1e-15;
  y1 = 1 + 1e-15;
  z1 = 1;
  w = fma(x, y, z);
  printf("%g\n", w);
}
