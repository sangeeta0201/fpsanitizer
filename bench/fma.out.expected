(output
  (function "main")
  (filename "fma.c")
  (line-num 11)
  (instr-addr 4006CA)
  (avg-error 0.000000)
  (max-error 0.000000)
  (num-calls 1)
  (influences
    (
    )
  )
)
